File: 30 Days of Code/Day 6 Let Review/Solution.java
import java.lang.StringBuilder;
import java.util.Scanner;

public class Solution {
    public static void main( String[ ] args ) {
        Scanner scanner = new Scanner( System.in );
        int t = scanner.nextInt( );
        StringBuilder bufferEvens = new StringBuilder( );
        StringBuilder bufferOdds = new StringBuilder( );

        scanner.nextLine( );
        for( int count = 1; count <= t; count++ ) {
            String word = scanner.nextLine( );

            for( int idx = 0; idx < word.length( ); idx++ ) {
                if( idx % 2 == 0 ) {
                    bufferEvens.append( word.charAt( idx ) );
                } else {
                    bufferOdds.append( word.charAt( idx ) );
                }
            }

            System.out.println( bufferEvens.toString( ) + " " + bufferOdds.toString( ) );
            bufferEvens.setLength( 0 );
            bufferOdds.setLength( 0 );
        }

    }
}

File: 30 Days of Code/Day 6 Let Review/Solution.cs
using System;
using System.Linq;
using System.Text;

public class Solution {
    private static bool isEven( int number ) => number % 2 == 0;

    public static void Main( string[ ] args ) {
        int t = Convert.ToInt32( Console.ReadLine( ) );
        StringBuilder bufferEvens = new StringBuilder( );
        StringBuilder bufferOdds = new StringBuilder( );

        for( int count = 1; count <= t; count++ ) {
            string word = Console.ReadLine( );

            for( int idx = 0; idx < word.Length; idx++ ) {
                if( isEven( idx ) ) {
                    bufferEvens.Append( word[ idx ] );
                } else {
                    bufferOdds.Append( word[ idx ] );
                }
            }
            Console.WriteLine( bufferEvens.ToString( ) + " " + bufferOdds.ToString( ) );
            bufferEvens.Clear( );
            bufferOdds.Clear( );
        }
    }
}